docs/MIGRATION_GUIDE.md

```diff
@@ -47,69 +47,46 @@ const client = new Client(config);
-    const query = productNodeQuery(['title', ['variants', variantConnectionQuery(['price', 'title'])]]);
-    ```
-    If the `query` argument isn't supplied, a default list of arguments will be used.
-    See the [API reference](https://github.com/Shopify/js-buy-sdk/blob/v1.0beta/docs/API_REFERENCE.md#Client.Queries) for more details.
-
-3.  Collections can be fetched with products using `fetchCollectionWithProducts()` and `fetchAllCollectionsWithProducts()`,
-    or by specifying products in the query function parameter when using `fetchCollection()` and `fetchAllCollections()`.
-    ```js
-    const collectionId = 'Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzI1NzY5NzczMQ=='
-    
-    // Use the built-in function
-    client.fetchCollectionWithProducts(collectionId).then((collection) => {
-      console.log(collection); // Collection with all default fields and products with all default fields.
-      console.log(collection.products); // Products on the collection
-    });
-
-    // or specify a custom query
-    const query = collectionNodeQuery(['title', ['products', productConnectionQuery(['title'])]]);
-
-    client.fetchCollection(collectionId, query).then((collection) => {
-      console.log(collection); // Collection with only title and products with only title
-      console.log(collection.products); // Products on the collection
-    });
-    ```
-    Note that `fetchCollectionWithProducts()` and `fetchCollectionsWithProducts()` do not take a `query` argument.
-
-4.  `fetchQueryProducts()` and `fetchQueryCollections()` query different fields and take an optional `query` argument.
-    See the [API reference](https://github.com/Shopify/js-buy-sdk/blob/v1.0beta/docs/API_REFERENCE.md) for more details.
-    
-    **v0.7:**
-    ```js
-    client.fetchQueryProducts({collection_id: '336903494', tag: ['hats']}).then((products) => {
-      console.log(products); // An array of products in collection '336903494' having the tag 'hats'
-    });
-    ```
-    
-    **v1:**
-    ```js
-    const queryObject = {updatedAtMin: '2016-09-25T21:31:33', sortBy: 'title'};
-    const query = productConnectionQuery(['updatedAt', 'title']);
-    
-    client.fetchQueryProducts(queryObject, query).then((products) => {
-      console.log(products); // An array of products with only updatedAt and title fields updated
-                             // after 2016-09-25T21:31:33 and sorted in ascending order by title
-    });
-    ```
-    
+1. The v1 functions take in a `Storefront ID` for fetching a product or collection by ID. A `Storefront ID` can be found under
+[the retrieving IDs section](https://help.shopify.com/api/storefront-api/getting-started#retrieving-ids) of the Storefront API docs.
+[Getting Started Guide](https://help.shopify.com/api/storefront-api/getting-started#authentication) for the Storefront API.
+
+2. Collections can be fetched with products using `collection.fetchWithProducts()` and `collection.fetchAllWithProducts()`.
```

Comment:
Worth explaining the diff between the two methods?

Reply:
the methods are self-documenting, but I should really be passing in an id to the first one so it's more clear 👍